back-end/helpers/user-helpers.js

```diff
@@ -18,6 +18,36 @@ async function getUserGitHubInfo(userID) {
   return userGitHubData;
 }
 
+async function getUserLanguage(userID) {
+  const query = 'SELECT language FROM github_info WHERE user_id = $1';
+  const userLanguage = await queryWithParameters(
+      query,
+      [userID]
+  );
+  return userLanguage;
```

Comment:
Return userLanguage.language here to avoid needing to have two variables in line 31 and 32.

Reply:
Updated to avoid the two variables -- you're right that userLanguage can return null or undefined but I believe this is only if the user hasn't linked their GitHub. Should I check for this on the back-end in this route or is this something we should do front-end?

Reply:
Or is it probably best to prevent any invalid DB query?

Reply:
For happy path, frontend can't send id/recommendations because they'll be sent to edit their profile instead.

However, it is a good idea to return a 4xx response back if the github info hasn't been attached yet.

Reply:
Sounds good, updated